subsystems/DrivetrainBase.java

```diff
@@ -75,13 +136,165 @@ public void tankDriveVoltsReverse(double leftVolts, double rightVolts) {
         this.tankDriveVolts(-leftVolts, -rightVolts);
     }
 
+    // NOTE: many of the following methods are self-explanatory. Only the neccesary documentation will be made.
+    public double getLeftPositionRotation() {
+        return leftLeader.getSelectedSensorPosition();
+    }
+
+    public double getRightPositionRotation() {
+        return rightLeader.getSelectedSensorPosition();
+    }
+
+    private double rotationsToMeters(double rotations) {
+        // number of rotations * circumference of wheel
+        return rotations * (double) constants.get("WHEEL_DIAMETER_METERS") * Math.PI;
+    }
+
+    public double getLeftPositionMeters() {
+        return rotationsToMeters(getLeftPositionRotation());
+    }
+
+    public double getRightPositionMeters() {
+        return rotationsToMeters(getRightPositionRotation());
+    }
+
+    public double getLeftVelocityRPM() {
+        return leftLeader.getSelectedSensorVelocity() / 2;
```

Comment:
Remove any dividing by 2 with the encoders, that's a sparkMax thing. Just put a `//TODO removed / 2, check if correct`, so we remember to test if I'm actually right.